src/app/clusters/client-monitoring-server/client-monitoring-server.cpp

```diff
@@ -19,20 +19,63 @@
 
 #include <app-common/zap-generated/af-structs.h>
 #include <app-common/zap-generated/attributes/Accessors.h>
+#include <app/AttributeAccessInterface.h>
 #include <app/CommandHandler.h>
+#include <app/ConcreteAttributePath.h>
+#include <app/util/ClientMonitoringRegistrationTable.h>
 #include <app/util/af-event.h>
 #include <app/util/af.h>
 #include <app/util/attribute-storage.h>
 
 using namespace chip;
+using namespace chip::app;
 using namespace chip::app::Clusters;
 using namespace chip::app::Clusters::ClientMonitoring;
 
+namespace {
+
+/**
+ * @brief Attribute Override class for ClientMonitoring cluster attribures
+ */
+class ClientMonitoringAttributeAccess : public app::AttributeAccessInterface
+{
+public:
+    ClientMonitoringAttributeAccess() : AttributeAccessInterface(Optional<EndpointId>(0), ClientMonitoring::Id) {}
+
+    CHIP_ERROR Read(const ConcreteReadAttributePath & aPath, AttributeValueEncoder & aEncoder) override
+    {
+        VerifyOrDie(aPath.mClusterId == ClientMonitoring::Id);
+
+        switch (aPath.mAttributeId)
+        {
+        case ClientMonitoring::Attributes::ExpectedClients::Id:
+            // TODO : Implement Client monitoring resgitration table
+            return CHIP_ERROR_NOT_IMPLEMENTED;
+
+        default:
+            break;
```

Comment:
if the attribute id is unknown, should the read return CHIP_NO_ERROR? Maybe we should set an error instead.

Reply:
By adding an error in the default section, all read commands for attributes not defined in the `AttributeAccessInterface` fail.